integration-tests/oidc-code-flow/src/test/java/io/quarkus/it/keycloak/CodeFlowTest.java

```diff
@@ -112,6 +125,7 @@ public void testTokenTimeoutLogout() throws IOException, InterruptedException {
     }
 
     @Test
+    @Disabled
```

Comment:
Any reason why this and other tests are marked as `@Disabled` ?

Reply:
@pedroigor Forgot to re-enable, the timeout test was failing so I was looking at that test only, sorry